### PR TITLE
2280 add quick summary of num missions, distance, labels, validations, and accuracy to dashboard

### PR DESCRIPTION
**Pull request by jonfroehlich**
Addresses https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2280 (and parts of the very old https://github.com/ProjectSidewalk/SidewalkWebpage/issues/1363) by:

- Adding the number of completed missions, total distance audited, num labels, num validations, and user accuracy to dashboard
- Removing the old version of these summary stats (which was in a table rather than an iconified design)
- Removing the the visualization of audit activity (which I didn't consider useful or important). If we want a historical visualization of activity, I suggest making a new Issue and developing one around labels or missions vs audits
- Removed the table of previous activity. I didn't consider this important either and I'm starting to make room for new user dashboard changes like the badges, etc. (See https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2281 and https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2288, for example).

### Old
image

### New
image

### Old
image

### New 
image

## Other
I also tested it as a brand new user and the dashboard properly showed me zeros and N/As, as appropriate. Here's a screenshot after validating 10 things (and completing 1 mission):

image

### CODE REVIEW

**Comment by misaugstad** [comment]
Love it!!! So exciting

**Comment by jonfroehlich** [comment]
Since we are no longer graphing a timeline and doing other things, we may be able to get rid of some of the many JavaScript imports (none of which are commented, so I don't know what they're associated with in the code). 

```
<script type="text/javascript" src='@routes.Assets.at("javascripts/lib/d3.v3.js")'></script>
<script type="text/javascript" src='@routes.Assets.at("javascripts/lib/c3.min.js")'></script>
<script type="text/javascript" src='@routes.Assets.at("javascripts/lib/moment/moment.js")'></script>
<script type="text/javascript" src='@routes.Assets.at("javascripts/lib/moment/es.js")'></script>
<script type="text/javascript" src='@routes.Assets.at("javascripts/lib/underscore-min.js")'></script>
<script type="text/javascript" src='@routes.Assets.at("javascripts/Progress/build/Progress.js")'></script>
<script type="text/javascript" src='@routes.Assets.at("javascripts/lib/i18next.min.js")'></script>
<script type="text/javascript" src='@routes.Assets.at("javascripts/lib/i18nextXHRBackend.min.js")'></script>
```

We need to keep the LabelMap related code, of course, which is here:

```
<script type="text/javascript" src='@routes.Assets.at("javascripts/SVLabel/src/SVLabel/util/Utilities.js")'></script>
<script type="text/javascript" src='@routes.Assets.at("javascripts/SVLabel/src/SVLabel/util/UtilitiesSidewalk.js")'></script>
<script type="text/javascript" src='@routes.Assets.at("javascripts/SVLabel/src/SVLabel/util/UtilitiesColor.js")'></script>
<script type="text/javascript" src='@routes.Assets.at("javascripts/lib/turf.min.js")'></script>
```

**Comment by misaugstad** [comment]
Related to that, I just took a quick peek. And you removed the HTML elements for the audit count chart and the missions list, but you didn't remove the JavaScript that filled in that chart and table in Progress.js. We need to delete all that code that we're no longer using. And I believe that both the audit count chart and the missions list had endpoints created to get that data, so we should be able to delete all the code controller and model code that is used only for those things.

**Comment by misaugstad** [comment]
And for those imports, I imagine that we can probably get rid of d3, c3, and moment. But the i18next imports are for Spanish translations, and Progress includes all the JavaScript code for the page.

Underscore just adds nice functional programming helper functions, but I think the only place we happen to be using them right now is for the missions list chart, so that can probably go too.

**Comment by jonfroehlich** [comment]
TBH, `Progress.js` is such a poorly named file and has no file-level header comments, so I just ignored it. I figured you'd either (1) delete the archaic code yourself or (2) tell me what to delete given your greater familiarity with the codebase.

**Comment by misaugstad** [comment]
Agree that Progress.js should be renamed to UserDashboard.js or the like. But I do think that any time an HTML element is deleted, we should be searching the entire project for the div IDs and associated CSS class names and deleting the associated CSS/JS that isn't used anywhere else. I can still do that for this PR, but I'm mostly saying it because it is easy for either of us to forget to do that when coding or reviewing, so if both of us are trying to remember to do that, hopefully at least one of us remembers every time it happens :upside_down_face: 

I should also add that to the PR template!

**Comment by jonfroehlich** [comment]
Sure. Totally agree. Let me know if you want me to do it. Otherwise, I'm gonna focus on https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2281 with the goal of creating a PR tomorrow. As mentioned during our 1:1, I won't have as much time the rest of the quarter to dev as heavily as I've been the past 1.5 weeks.

So, trying to push on https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2281 now that I've created this PR.

**Comment by misaugstad** [comment]
Yep, please go for it! Just know that it'll take me some time to get through all the open PRs you made :grin: but it'll all get closed out in the coming weeks

**Comment by jonfroehlich** [comment]
Gotcha. Hopefully this one won't take a couple of weeks as it's relatively straightforward and you helped with the backend stuff. Most of the heavy lifting here is just in coming up with a clean design, making the icons, etc. I think it would be nice to merge this PR alongside @rpechuk's leaderboard work as they complement each other.

**Comment by jonfroehlich** [comment]
Also note that we still have this (for when the user is a turker):

```
@* Only show this table if the current user is a turker *@
        @if(user && user.get.role.getOrElse("") == "Turker") {
            <div class="row" style="margin-top: 20px">
                <table class="table table-striped table-condensed">
                    <tr>
                        <th>@Messages("dashboard.distance")</th>
                        <th>@Messages("dashboard.missions")</th>
                        <th>Total Reward Earned</th>
                    </tr>
                    <tr>
                        <!-- The id='td-total-distance-audited' is automatically filled by "javascripts/Progress/build/Progress.js"-->
                        <td id="td-total-distance-audited">Hello</td>
                        <td>@MissionTable.countCompletedMissionsByUserId(user.get.userId, includeOnboarding = false)</td>
                        <td id="td-total-reward-earned"></td>
                    </tr>
                </table>
            </div>
        }
```

**Comment by misaugstad** [comment]
No I don't think there's a reason they would need/want to see that.

**Comment by jonfroehlich** [comment]
Ok. Good. Then I think we are ready again for the PR.

**Comment by misaugstad** [comment]
Cool. Reminder to myself when I look at this to remove the routes/controller/model code associated with the ajax calls removed from Progress.js.

File: public/stylesheets/userProfile.css [hunk 1]
```diff
@@ -79,4 +79,30 @@ svg {
...
+    height:70px;
+    margin: auto; /* centers the image horizontally */
+}
+
+.primary-user-stats-holder-stat-header {
+    font-size: 30px;
+    font-family: raleway-extrabold, sans-serif;
+}
+
+.ps-skyline-stats-holder-stat-number {
```
Comment (misaugstad) [inline public/stylesheets/userProfile.css:103]: Fixed the name of this and just updated the names of some other CSS in here.

File: app/views/userProfile.scala.html [hunk 2]
```diff
@@ -24,58 +26,50 @@
...
+        
+        <div class="row" style="margin-top: 30px">
+            <div class="primary-user-stats-holder">
+                <span class="primary-user-stats-holder-stat-header">@Messages("dashboard.your.missions")</span>
+                <span class="primary-user-stats-holder-stat-header">@Messages("dashboard.distance")</span>
+                <span class="primary-user-stats-holder-stat-header">@Messages("dashboard.labels")</span>
+                <span class="primary-user-stats-holder-stat-header">@Messages("dashboard.validations")</span>
+                <span class="primary-user-stats-holder-stat-header">@Messages("dashboard.accuracy")</span>
+
+                <img src='@routes.Assets.at("images/icons/project_sidewalk_flag.png")'/>
```
Comment (misaugstad) [inline app/views/userProfile.scala.html:38]: Adding alt text to the images for screen readers and such

File: app/views/userProfile.scala.html [hunk 3]
```diff
@@ -24,58 +26,50 @@
...
-                </tr>
-                <tr>
-                    <td id="td-total-distance-audited"></td>
-                    <td>@MissionTable.countCompletedMissionsByUserId(user.get.userId, includeOnboarding = false)</td>
-                    <td id="td-total-reward-earned"></td>
-                </tr>
-            </table>
-        </div>
-        <div class="row">
-            <table class="table table-striped table-condensed">
```
Comment (misaugstad) [inline app/views/userProfile.scala.html:47]: Some of the CSS from removed HTML elements can be removed too

**Review by misaugstad — COMMENTED** [review]
I'm going to add comments throughout the code as I make changes just so that you know what I've been changing, but I'll likely just do most/all of it myself instead of asking you to fix stuff.

**Review by misaugstad — APPROVED** [review]

**Pull request merged**